app/src/main/java/com/epfl/drawyourpath/mainpage/fragments/runStats/ShapePathDescriptionFragment.kt

```diff
@@ -29,6 +30,6 @@ class ShapePathDescriptionFragment(private val formName: String, private val sco
         this.formDescriptionText = view.findViewById(R.id.formDescriptionPath)
         this.formDescriptionText.text = "${getString(R.string.shape_recognized_on_the_path_drawn)} $formName"
         this.scoreText = view.findViewById(R.id.scorePath)
-        this.scoreText.text = "${getString(R.string.score_of_the_shape_recognized)} $score"
+        this.scoreText.text = "${getString(R.string.score_of_the_shape_recognized)} ${String.format("%.2f", score)}"
     }
```

Comment:
Maybe we can convert this score in percent and indicate % at the end of the score line .

Reply:
It is hard to do a percentage as the best score you can get is 0, and it can show positive or negative values of an unknown range. [here is the documentation](https://developers.google.com/android/reference/com/google/mlkit/vision/digitalink/RecognitionCandidate#getScore())